src/api_server/swagger/firecracker.yaml

```
@@ -604,7 +604,8 @@ paths:
     get:
       summary: Gets the full VM configuration.
       description:
-        Gets configuration for all VM resources.
+        Gets configuration for all VM resources. If the VM is restored from a snapshot the boot-source,
+        machine-config.ht_enabled, and machine-config.cpu_template will all have default values.
```

```suggestion
        Gets configuration for all VM resources. If the VM is restored from a snapshot, the boot-source,
        machine-config.ht_enabled and machine-config.cpu_template will be empty.
```

They should be empty, since they don't really have default values

---

I figured this comment meant that boot-source should be null when a snapshot is restored instead of a default full object value. Is that what you meant? I made boot-source null in this [commit](https://github.com/firecracker-microvm/firecracker/pull/2806/commits/3d652cc2d8af7b396e0140812172277976bb7ad6).